CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.18.0] - 2022-12-dd
+
```

Comment:
Like in other releases there are groups, I would put both of these under `Changed`

Reply:
This was somehow an overview from my side... I added a changed header